frontend/packages/dev-console/src/components/topology/shapes/WorkloadNode.tsx

```diff
@@ -62,6 +70,39 @@ const WorkloadNode: React.FC<NodeProps<WorkloadData>> = ({
             </g>
           </Decorator>
         ),
+        build && (
+          <Link
+            to={`/k8s/ns/${build.metadata.namespace}/buildconfigs/${
+              build.metadata.ownerReferences[0].name
```

Comment:
use the utilities from `resource-link` to construct the link.

Reply:
fixed in https://github.com/openshift/console/pull/2193/commits/2160ad48ba2a1561104dc6e47e77d441e3a7b1cf